src/phylogeny.rs

```diff
@@ -0,0 +1,15 @@
+// Copyright 2020 Franklin Delehelle
+// Licensed under the MIT license (http://opensource.org/licenses/MIT)
+// This file may not be copied, modified, or distributed
+// except according to those terms.
+
+//! A phylogenetic tree is represented as a directed graph.
+//! Each node is a taxon, identified as a string.
+//! The edges are weighted by the phylogenetic distance if it was defined, or f32::NAN otherwise.
+
+use petgraph::graph::Graph;
+
+pub type Taxon = String;
+pub type Proximity = f32;
+
+pub type Tree = Graph<Taxon, Proximity>;
```

Comment:
Shouldn't this also be fixed to a directed graph by default, as is mentioned in the explanatory comment above?

```suggestion
pub type Tree = Graph<Taxon, Proximity, Directed>;
```

This should also require a `use petgraph::Directed;`.

In looking at this, I came across the petgraph_examples crate, that sported this example:
https://docs.rs/petgraph-examples/0.1.0/src/petgraph_examples/lib.rs.html#57-69

Reply:
`Graph` aliases to a directed graph by default, [cf. here](https://docs.rs/petgraph/0.5.1/src/petgraph/graph_impl/mod.rs.html#332-336). I guess we could be more explicit, but I assumed that people would presume that phylo trees are necessarily directed.

Reply:
I think I'd prefer `petgraph::Directed` as well. I also wonder whether this should probably be a newtype instead of just a type alias.